cds_dojson/marc21/fields/utils.py

```diff
@@ -413,7 +413,7 @@ def build_contributor_books(value):
         'full_name': value.get('name') or clean_val('a', value, str),
         'roles': [
             _get_correct_books_contributor_role(
-                'e', value.get('e', 'author')).lower()
+                'e', value.get('e', 'author'))
```

Comment:
we need the `.lower()` in case there are different versions like `ed.` and `Ed.` `Ed.` etc

Reply:
this check if done here https://github.com/CERNDocumentServer/cds-dojson/blob/2cd5bb537775998a7c89813cb14d085d0f2fc8ba/cds_dojson/marc21/fields/utils.py#L337